iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -282,7 +282,7 @@ public void forceUpdateNodeCache(
    */
   public void removeNodeCache(int nodeId) {
     loadCache.removeNodeCache(nodeId);
-    loadCache.updateNodeStatistics();
+    loadCache.updateNodeStatistics(false);
```

Comment:
Why set here to false?

Reply:
It looks like this is also an active update, not a heartbeat update, maybe set it to true?